tests/test_diffraction_objects.py

```diff
@@ -193,14 +156,9 @@ def dicts_equal(dict1, dict2):
 
 @pytest.mark.parametrize("inputs1, inputs2, expected", params)
 def test_diffraction_objects_equality(inputs1, inputs2, expected):
-    diffraction_object1 = DiffractionObject(**inputs1)
-    diffraction_object2 = DiffractionObject(**inputs2)
-    # diffraction_object1_attributes = [key for key in diffraction_object1.__dict__ if not key.startswith("_")]
-    # for i, attribute in enumerate(diffraction_object1_attributes):
-    #     setattr(diffraction_object1, attribute, inputs1[i])
-    #     setattr(diffraction_object2, attribute, inputs2[i])
-    print(dicts_equal(diffraction_object1.__dict__, diffraction_object2.__dict__), expected)
-    assert dicts_equal(diffraction_object1.__dict__, diffraction_object2.__dict__) == expected
```

Comment:
Since we will be introducing an unique `uuid` via `_id`, this test will fail. But rather we use `__eq__` method.

Reply:
@bobleesj yes, this is much cleaner.  A sure sign that our code and API is improving, don't you think?

Reply:
absolutely